### PR TITLE
User completed (for now)

### PR DESCRIPTION
**Pull request by Stanislaw09**

### CODE REVIEW

File: src/components/user/menuDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      _id: string,
+      name: string,
+      email: string
+   },
+   setOpen: (val: boolean)=>void,
+   setAlert: (val: boolean)=>void,
+   editUser: (val: {name: string, email: string})=>void
+})=>{
+   const classes=useStyles()
+   const [username, setUsername]=useState<string>(user.name)
```
Comment (arleta-jedrzejczak) [inline src/components/user/menuDialog.tsx]: please, add prettier to the project

File: src/components/user/newPostDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      display: 'flex',
+      justifyContent: 'space-around'
+   },
+   imagePreview:{
+      maxHeight: '50vh',
+      objectFit: 'cover'
+   }
+}))
+
+export const NewPostDialog=({open, setOpen, setPosts, user}: {
```
Comment (arleta-jedrzejczak) [inline src/components/user/newPostDialog.tsx]: please add interface to NewPostDialog in separate file and use it here

File: src/components/user/user.tsx [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+            _user._id===id && setUser(_user)
+         });
+      }).catch(err=>console.log(err))
+   }, [])
+
+   const handleMenuOpen =(e: React.SyntheticEvent)=> setMenuAnchor(e.currentTarget)
+
+   const handleMenuClose = () => setMenuAnchor(null)
+
+   const handleTabChange=(e: React.SyntheticEvent, value: number)=>setTabValue(value)
```
Comment (arleta-jedrzejczak) [inline src/components/user/user.tsx]: if 'e' isn't used, you can use '_' instead

**Review by arleta-jedrzejczak — CHANGES_REQUESTED** [review]

File: src/components/user/menuDialog.tsx (hunk 1 above)
Comment (Stanislaw09) [reply src/components/user/menuDialog.tsx]: with heartache but I did it

File: src/components/user/newPostDialog.tsx (hunk 2 above)
Comment (Stanislaw09) [reply src/components/user/newPostDialog.tsx]: I've created new file with all my interfaces

**Review by arleta-jedrzejczak — APPROVED** [review]

**Pull request merged**